_docs/tutorial/05-adding-a-new-version-of-a-microservice.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Adding a new version of a microservice
+overview: Overview
+
+order: 05
+
+layout: docs
+type: markdown
+---
+{% include home.html %}\n# Deploy and release a new version of _reviews_
+
+Let's deploy a new version of the _reviews_ microservice, the one that will return the ratings provided by reviewers, as a number of stars, with the color of stars. In the real life, we would perform lint tests, unit tests, integration tests, end-to-end tests and tests in the staging environment.
+
+1. Deploy the new version of the _reviews_ microservice without the `app` label. Without the `app` label, our new version of the microservice will not be selected to provide the _reviews_ service. As such, it will not be called by the production code.
+  ```bash
+  kubectl apply -f bookinfo-reviews-v2-without-app-label.yaml
+  ```
+
+2. Let's access our application and see that the deployed microservice did not disrupt it. So far so good.
+
+3. Now let's test the new version of our microservice from inside the cluster. We will use the `sleep` container we deployed earlier. Note that our new version hits the production pods of the _ratings_ microservice during the test. Also note that we have to access our new version of the microservice by its pod IP, since it is not selected for the _reviews_ service.
+
+  1. Get the IP of the pod:
+     ```bash
+     DETAILS_V2_POD_IP=$(kubectl get pod -l version=v2 -o jsonpath='{.items[0].status.podIP}')
+     ```
+  2. Send a request to the pod and see that it returns the correct result:
+    ```bash
+    kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') -- curl $DETAILS_V2_POD_IP:9080/reviews/7
+    ```
+  3. Perform a primitive "load testing" - send requests for 10 times in a row:
```

Comment:
Perform a primitive "load testing" -> Perform primitive load testing
requests for 10 -> requests 10